testing/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/configurationcache/ConfigurationCacheFixture.groovy

```diff
@@ -475,6 +480,10 @@ class ConfigurationCacheFixture {
             changedTask = name
         }
 
+        void pluginInputChanged(String name) {
```

Comment:
💭 Introduced with the original test verifying the broken behavior. I decided to keep it in case we need it again.